todo.md

```diff
@@ -14721,458 +14721,41 @@ into your web browser:
 ## zh_CN
 
 <details>
-<summary>show<small> (all missing: 144, including not present: 18)</small></summary>
+<summary>show<small> (all missing: 3, including not present: 0)</small></summary>
 
- * json : A new verification link has been sent to the email address you provided during registration.
- * json : Accept Invitation : not present
- * json : Add a new team member to your team, allowing them to collaborate with you.
- * json : Add additional security to your account using two factor authentication.
- * json : Add Team Member
- * json : Add
- * json : Added.
- * json : Administrator users can perform any action.
- * json : Administrator
- * json : All of the people that are part of this team.
- * json : Already registered?
- * json : API Token Permissions
  * json : API Token
```

Comment:
If it's same translation, add it in exclude.php

Reply:
OK